src/lib.jl

```diff
@@ -516,3 +515,35 @@ Python function.
 function entropic_gromov_wasserstein(μ, ν, Cμ, Cν, ε, loss="square_loss"; kwargs...)
     return pot.gromov.entropic_gromov_wasserstein(Cμ, Cν, μ, ν, loss, ε; kwargs...)
 end
+
+"""
+    mm_unbalanced(a, b, M, reg_m; kwargs...)
```

Comment:
I think it could be useful to mention the `reg` and `c` keyword arguments since they appear in the formula below.
```suggestion
    mm_unbalanced(a, b, M, reg_m; reg=0, c=a*b', kwargs...)
```